code/modules/uplink/uplink_item.dm

```diff
@@ -914,8 +914,8 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	cost = 3
 
 /datum/uplink_item/device_tools/military_belt
-	name = "Chest Rig"
-	desc = "A robust seven-slot set of webbing that is capable of holding all manner of tatical equipment."
+	name = "Military Belt"
```

Comment:
yes I said that I am aware that this gives the belt back to traitors so ops can have their webbing